python/doc/examples/meta_modeling/viscous_fall_metamodel.ipynb

```diff
@@ -93,134 +155,261 @@
BWxz6e28CFcBK4AMwJJ63sGAYORiBD0N4xxoYORKqVcN1fE26lQ2j1wa4+jS5Byb8ytJdXYnXr7UFyHPkyBHteI4cOT6fe/36NdmzZ492XqVSYWpq+vma2ISHhxMe/qW2ZWCgtk5kZGQkkZHJmxU2a2QOJKRkbzc9+v8zEs/qx8Szir+EPKv+06GYg8SUHko6loNpW6KwPQUvhyh42kNJwAkNuRaoUWRK6V7rRoK+r3LqI7k7oOp+hqj6uZCbWif6vsNYgKXChoXK0TzXPGCMegX6GCS6ve+quhbVX9XgQAOiGp4Bg2xYYowXA3FWzaMGs/GKGkgusgJgBcyUJHqrVDSKiqLup38FfO9ZZWcSkUo/nkguIBtjvGoVqrJlmf9XB/6130qzxRLnh0eR7RfZF5iSP6/Ez8B0GACmpGnTpjFx4sQYxw8fPkzGjBlT5J5eXvFZRSKAeFYJIZ5V/MX7WRlAlz8zsW26PV0qZKR+n38o3sAHw8y50Cwuid/JMN4Nv0xUnvhXxUhr4v2szGTsHfQx7XGC46FmhJsoE31PU4rQPOdw9pSdy73Am7S6OIqMkcaJbu97DFUDqBo4nODtTpzLOgmNpAfAKMPijK18kUqaaUw5WwGzMG0QagGUrlCBLsbGeBw/jtFXQUecz0qqj2m5OzzO0QY/n/GYurri8OefTLb8nTaqcdSdFciE8udRKn6d18op8fMqJETkbZJkOW1PTkiSxK5du2jSpAkAjx49wtbWluvXr1OqVKnP11WrVo1SpUoxb948PD09GTJkCB8+fMnqHhUVhYGBAdu2baNp06ax3iu2N4C5c+fm7du3GBsn7w+cyMhIvLy8cHZ2Rk9PL1nbTm/Es4o/8aziL7HPKjQYZvRTcnCDgma91AyapUH9CJ62URHxGHItVJO1Q5r+sRtDop6VXxiq0ruQHbKj3l4TJClJfbgpXWC4shmZMWF21F5ykz9J7cVFenMR5UEnZOuWqKus/Nzvx7zFWTWPDKg4EjUQS0wA8AFKq1Q0kmVWqtXxelYawniibEyIdBXbqCNk6rscxfr1HNl+k3r78jGwhoY/m6ZwUuxUICV/XgUGBmJubk5AQECy//5OK9LdG0AbGxssLCw4evTo5wAwMDCQixcv0rt3bwAcHR3x9/fn6tWrlC1bFoBjx46h0WhwcHCIs219fX309fVjHNfT00uxX6Yp2XZ6I55V/IlnFX8JfVZ6JjB5HZSuArMGKHlwQ8mf26DIZXjWB553UxF6AXLPA0UKzVbqSoKelaUeLKmC1MwLxdan0KFAku5dhiqs5gID+A1Xvaq4s5cSOCapzVhZVYaqK5GOd0BhVhxKDQegIDk5zhCqM5vaevM5jhuWmGADzAW6ShItFQrqfmrm+89Kj/zs4T9q8livAYXmHcHg9GmcRzdnxrQrDNmtooKtkpZlk394qVFK/LwSP//SaBqYoKAgbty4wY0bNwDtxo8bN27w7NkzJEli0KBBTJkyhb1793Lz5k06deqEpaXl57eEhQsXpm7duvTo0YNLly5x9uxZ+vXrR5s2bbC0tNTl0ARBSAckCZr3hOWn4c0L6FAGrl6AvKs+JY5eA/cqilrCNLWBdvmh/1l4GZzk5nJhiyfnsMaOXtTgCNuToZOxKNAeSo+BSyPhyZ7Ph/ORjeMMIYQIauLOK7RZnDsDDQBXiCV9dOyUZCY/f6MkKw8MGxO5dR7cvctgrxG0Lgdd18Cdl8k9MOFXkiYDwCtXrlC6dGlKly4NgJubG6VLl2bcuHEADB8+nP79++Pq6kr58uUJCgri4MGDGBh8+ef2hg0bsLOzo1atWvz2229UrlyZZcuW6WQ8giCkT8XsYf1VKFAC+jrDmhlg1g3szoM6AO6WAf+9uu6ljnlUBAMluJ5Olu3SJpixiCPUoBkjack6ZqXMDuFyk8CmGRxrD+/++XzYlmwcx40gwqnFHHwJRAKWAmHAEGX81zvqkY0CHEImggfFhxHlPg5pjjsrrY5iYwZNl0BAaLKPTPhFpMkAsHr16siyHONr9erVgHZd4KRJk3j9+jVhYWEcOXKEggULRmvD1NSUjRs38vHjRwICAvD09MTIyEgHoxEEIT3Lmg3mH4LOI2DBSBjREuT82lQxmWvAw8bwYjTIv2q1BzMDWFoF/noGa/5LliYzoM9k1tOVUcxjGDPoRxTJ/IAlBVRfAyaF4GBDCPmSQSI/2TmGG/6EUIs5vCEQS7RTwRsUCi59k6Xie/TJSwEOEcFzHvY5hKZBLTK5dGBnq/e8DoAuq0GT/pcDCikgTQaAgiAIaYlSCX3/gFm74OJh6GwPPq8h306wmgGvp8ODOhD5Rtc91ZFG1tCpgDY3oE/ylFBRoKAvfzCapexkKcNoSihJn2aORi8T1Nmrjd4PN4WoL9WmCpKDY7jxliCcmMtbgugE1NNoWFyqFB/ibjUGQ4qSn78Ilq7waJsKWY6kwMiurO8ms/sGTD+UvMMSfg0iABQEQfhJqjeBNZe1awQ728OJ3WAxDAoehdDb2inhoMTnR07b5lUEIz3ofipZM2c3wxV39nGVE7hSjbfEnes1UTJZQe3d8O4GnOoRre92WHCMwfgSiBNzeE8Qi9RqwpVKhiZgKhjACEds2UGAwVGeXiqFvHcvDS8s4/f6MHYPeN1J3mEJ6Z8IAAVBEH4i60Kw+iJUqAPDmsHC0ZCxChS+Bvo2cL8avJn/C1YPMdGHFVXhkA+sup+sTVeiHss5jR8v6UoFHnM3Wdsnuz1U8wTv9fDPjGinimDJUQbzAn+cmUtGQnC5dYt1CgV/JfA2WaiHNat4l+coLw+UgsGDGZ//HrWLQNsV8PRd8g1JSP9EACgIgvCTZcoMf26FATNgzXQYVB9CDaDgMcjeD54PgCedQPOr5aqtlwc6FwS3C/AieadrC1GK1VzAECNcqMQ1TiVr++RvC6XHwqVR8CT6zp5iWHGUwTzjPfWVC3F4+ZC6Gg2ukKCpYAAzOpALd17XvYH
```

Comment:
Github suggestion feature did not work:


In order to create a `Function` from this Python function, we use the `PythonPointToFieldFunction` class. Since the altitude is the only output field, the third argument `outputDimension` is equal to `1`. If we had computed the speed as an extra output field, we would have set `2` instead.
```

Reply:
Fixed.